<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleXYScanWindow</class>
 <widget class="QMainWindow" name="SimpleXYScanWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>577</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Simple XY Scan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0" columnstretch="0,0">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="control_groupBox">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>200</height>
           </size>
          </property>
          <property name="title">
           <string>XY Scan Control</string>
          </property>
          <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0">
           <item row="1" column="0" colspan="5">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="go_pushButton">
                 <property name="text">
                  <string>Go To</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="halt_pushButton">
                 <property name="text">
                  <string>Halt</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="previous_pushButton">
                 <property name="text">
                  <string>Previous</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="saved_y_doubleSpinBox"/>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="saved_x_label">
                 <property name="text">
                  <string>pos X</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="y_label">
                 <property name="text">
                  <string>current Y</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QDoubleSpinBox" name="saved_x_doubleSpinBox"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="x_label">
                 <property name="text">
                  <string>current X</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLabel" name="z_label">
                 <property name="text">
                  <string>current Z</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="saved_z_label">
                 <property name="text">
                  <string>pos Z</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QDoubleSpinBox" name="saved_z_doubleSpinBox"/>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="saved_y_label">
                 <property name="text">
                  <string>pos Y</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QDoubleSpinBox" name="x_doubleSpinBox"/>
               </item>
               <item row="3" column="1">
                <widget class="QDoubleSpinBox" name="y_doubleSpinBox"/>
               </item>
               <item row="3" column="2">
                <widget class="QDoubleSpinBox" name="z_doubleSpinBox"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="6" column="0" colspan="5">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="save_pushButton">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="position_name_lineEdit"/>
           </item>
           <item row="0" column="4">
            <widget class="QPushButton" name="delete_pushButton">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="position_groupBox">
            <property name="title">
             <string>Position List</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2"/>
             </item>
             <item>
              <widget class="QListWidget" name="position_listWidget"/>
             </item>
             <item>
              <widget class="QPushButton" name="load_file_pushButton">
               <property name="text">
                <string>Load from file</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="save_file_pushButton">
               <property name="text">
                <string>Save to file</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
